spec/rubocop/cop/style/empty_else_spec.rb

```diff
@@ -35,6 +35,18 @@
     end
   end
 
+  shared_examples_for 'offense registration' do
+    it 'registers an offense with correct message' do
+      inspect_source(cop, source)
+      expect(cop.messages).to eq(['Redundant `else`-clause.'])
+    end
+
+    it 'registers an offense with correct location' do
+      inspect_source(cop, source)
+      expect(cop.offenses.first.location).to be_a(Parser::Source::Range)
```

Comment:
Checking the type is a bit too whitebox IMO. I would prefer 
```ruby
expect(cop.highlights).to eq(['else'])
```
That would also catch the bug.